### PR TITLE
Fix layer implementation of vk_layerGetPhysicalDeviceProcAddr

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**
This function must be queryable from vkGetInstanceProcAddr() - see https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers/blob/master/loader/LoaderAndLayerInterface.md

This fixes the crash reported in https://github.com/KhronosGroup/Vulkan-Loader/issues/950

### CODE REVIEW

File: scripts/helper_file_generator.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def genCmd(self, cmdInfo, name, alias):
             'vkEnumerateInstanceVersion',
             'vkGetDeviceProcAddr',
             'vkGetInstanceProcAddr',
-            'vkGetPhysicalDeviceProcAddr',
+            'vk_layerGetPhysicalDeviceProcAddr',
```
Comment (mikes-lunarg) [inline scripts/helper_file_generator.py:322]: I think this line can be removed entirely, neither name is in the xml

**Review by charles-lunarg — APPROVED** [review]
This looks good.

File: scripts/layer_chassis_generator.py [hunk 2]
```diff
@@ -142,7 +142,7 @@ class LayerChassisOutputGenerator(OutputGenerator):
         # Include functions here to be interecpted w/ manually implemented function bodies
         'vkGetDeviceProcAddr',
         'vkGetInstanceProcAddr',
-        'vkGetPhysicalDeviceProcAddr',
+        'vk_layerGetPhysicalDeviceProcAddr',
```
Comment (mikes-lunarg) [inline scripts/layer_chassis_generator.py:145]: Also can be removed

File: scripts/dispatch_table_helper_generator.py [hunk 3]
```diff
@@ -294,6 +294,7 @@ def OutputDispatchTableHelper(self, table_type):
             table += 'static inline void layer_init_instance_dispatch_table(VkInstance instance, VkLayerInstanceDispatchTable *table, PFN_vkGetInstanceProcAddr gpa) {\n'
             table += '    memset(table, 0, sizeof(*table));\n'
             table += '    // Instance function pointers\n'
+            table += '    table->GetPhysicalDeviceProcAddr = (PFN_GetPhysicalDeviceProcAddr) gpa(instance, "vk_layerGetPhysicalDeviceProcAddr");\n'
```
Comment (mikes-lunarg) [inline scripts/dispatch_table_helper_generator.py:297]: Yikes, nice catch actually initializing this function pointer.

**Review by mikes-lunarg — APPROVED** [review]

**Pull request merged**